README.md

```diff
@@ -1 +1,13 @@
-# rocketmq-apis
+## Overview
+
+## Extension Spec
+
+## Message Model
+
+## Producer Flow
+
+## Consumer Flow
+
+## Reserved Fields
+
+This API spec has reserved the numeric value of PB enum to 64 and the field numeric value of PB message to 64. These reserved values will be used by RocketMQ's official implementation. Beyond the reserved values are free to use by external organizations.
```

Comment:
Messages and enumerations of the API spec reserve the first 64 fields for the evolution of RocketMQ exclusively. Vendor-specific extensions to the protocol are supposed to use fields beyond 64.